intro-HPC/ch_MATLAB.tex

```diff
@@ -0,0 +1,80 @@
+\chapter{MATLAB}
+\label{ch:matlab}
+
+\section{Why is the MATLAB compiler required?}
+
+The main reason behind this alternative way of using MATLAB is licensing: only
+a limited number of MATLAB sessions can be active at the same time. However, once
+the MATLAB program is compiled using the MATLAB compiler, the resulting stand-alone
+executable can be run without needing to contact the license server.
+
+\section{How to compile MATLAB code}
+
+Compiling MATLAB code can only be done from the login nodes.
+
+To access the MATLAB compiler, the MATLAB module should be loaded first:
+
+\begin{prompt}
+%\shellcmd{module avail MATLAB}%
+----------------------/apps/gent/CO7/sandybridge/modules/all ----------------------
+   MATLAB/2016b    MATLAB/2017b    MATLAB/2018a (D)
+%\shellcmd{module load MATLAB/2018a}
+\end{prompt}
+
+Then the \verb|mcc| command can be used. To compile a single file, the \verb|-m|
```

Comment:
Rephrase, starting with `Then` sounds weird. And maybe show the output of `which mcc`?

Also mention `mcc -?` to get help (yes, `-?`...)